source/operations/troubleshooting.rst

```diff
@@ -156,6 +158,41 @@ You can scroll vertically through the output for the selected check.
    :alt: MinIO SUBNET's health report for a deployment showing a failed Health Report with details expanded
    :align: center
 
+Logs
+----
+
+Use :mc:`mc admin logs` command to display logs from the command line.
+The command supports type and quantity filters for further limiting logs output.
+
+Optionally, use :ref:`Call Home <minio-troubleshooting-call-home>` to start automatically uploading real time error logs to SUBNET for analysis.
+
+.. _minio-troubleshooting-call-home:
+
+Call Home
+---------
+
+Call Home functionality automates the collection and uploading of diagnostic data or error logs to SUBNET.
+You much have an active SUBNET registration for the deployment to use Call Home.
+
+When enabled, Call Home can send one or both of:
+
+- error logs to SUBNET in real time
+- a new diagnostic report every 24 hours
+
+The diagnostic report upload happens every 24 hours from the time you enable Call Home.
+If needed, you can adjust the frequency of the diagnostic reports and uploads.
```

Comment:
I don't see this via mc inline help. is this / does this need to be documented?

Reply:
It is possible. Whether we want to document that it is possible is another thing.

Reply:
Adjusting the frequency was something that was implemented a while ago, but I am not sure if it is applicable to the current code.
I do not think we should document this as this could lead to confusion. It would just be safe to say that the upload happens "every 24 hours from the time minio server was started".

Reply:
@nagashreem Is it from the time of the server start or the time of enabling Call Home? My understanding is that it's form the time you enable Call Home.
@harshavardhana?

Reply:
From what I understand, it should be from the time of the server start. Will confirm.

Reply:
@anjalshireesh confirmed that the restart time is one of:
1. 24 hours since Call Home was enabled
2. 24 hours since the last deployment restart (all nodes) when Call Home was already enabled.

Adding to the doc.